test/test_examples.py

```diff
@@ -37,11 +37,15 @@ def name_generator(__ctr=[0]):
     __ctr[0] += 1
     return 'testfile_%d' % __ctr[0]
 
-def test_lltest(tmpdir):
+@pytest.fixture(params=['asyncio', 'trio'])
+def aio(request):
+    return request.param
+
+def test_hello(tmpdir, aio):
```

Comment:
You can just do

```
@pytest.mark.parametrize("name", ('hello.py', 'hello_asyncio.py'))
def test_hello(name):
#...
```

No need for the fixture

Reply:
Ah. Still new to pytest. Thanks. :-)